scripts/build_devcontainer_image.sh

```diff
@@ -1,48 +0,0 @@
-#!/bin/bash
-#
-# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
-#
-# NVIDIA CORPORATION and its licensors retain all intellectual property
-# and proprietary rights in and to this software, related documentation
-# and any modifications thereto.  Any use, reproduction, disclosure or
-# distribution of this software and related documentation without an express
-# license agreement from NVIDIA CORPORATION is strictly prohibited.
-ROOT="src/isaac_ros/isaac_ros_common/scripts"
-source $ROOT/utils/print_color.sh
-
-PLATFORM="$(uname -m)"
-BASE_NAME="isaac_ros_dev-$PLATFORM"
-CONTAINER_NAME="$BASE_NAME-container-$USER"
-
-# Read and parse config file if exists
-#
-# CONFIG_IMAGE_KEY (string, can be empty)
-if [[ -f "${ROOT}/.isaac_ros_common-config" ]]; then
-    . "${ROOT}/.isaac_ros_common-config"
-fi
-
-# Build image
-IMAGE_KEY=ros2_humble
-if [[ ! -z "${CONFIG_IMAGE_KEY}" ]]; then
-    IMAGE_KEY=$CONFIG_IMAGE_KEY
-fi
-
```

Comment:
@jmblixt3 Why are we deleting this file? How does deleting this affect our dev container? (Asking because I genuinely don't know)

Reply:
I replaced this, with a new file